### PR TITLE
Add remote browser feature

### PR DESCRIPTION
**Pull request by William-Yeh**
This feature exists in Ruby version of Capybara.

### CODE REVIEW

File: capybara/selenium/browser.py [hunk 1]
```diff
@@ -28,5 +28,7 @@ def get_browser(browser_name, capabilities=None, **options):
         return webdriver.PhantomJS(desired_capabilities=capabilities, **options)
     if browser_name == "safari":
         return webdriver.Safari(desired_capabilities=capabilities, **options)
+    if browser_name == "remote":
```
Comment (elliterate) [inline capybara/selenium/browser.py]: Can you sort this alphabetically by placing it above the Safari code block?

**Review by elliterate — CHANGES_REQUESTED** [review]
Thanks for opening this request! Just one minor nit.

**Review by William-Yeh — COMMENTED** [review]
sure!

**Review by William-Yeh — COMMENTED** [review]
sure!

**Review by elliterate — APPROVED** [review]

**Pull request merged**